scripts/configure.js

```diff
@@ -136,6 +143,20 @@ function lldbVersionToBranch(version) {
 
 // On Mac the lldb version string doesn't match the original lldb versions.
 function getDarwinRelease() {
+  try {
+    const version = child_process.execFileSync('llvm-config', ['--version']).toString().trim();
+    const prefix = child_process.execFileSync('llvm-config', ['--prefix']).toString().trim();
+    const options = JSON.stringify({
+      variables: { "lldb_build_dir%": prefix }
+    }, null, 2);
+    fs.writeFileSync('options.gypi', Buffer.from(options), 'utf-8');
+    console.log('Overwriting options.gypi with output from llvm-config:');
+    console.log(options);
+    return version.split('.').slice(0, 2).join('.');
+  } catch(err) {
```

Comment:
This catch block should arguably be just for the `.execFileSync()` calls.  Rule of thumb: catch blocks should be no bigger than absolutely necessary, particularly catch-and-swallow blocks.

Reply:
Good catch. Will fix that, thanks